packages/legend-studio/src/components/editor/side-bar/Explorer.tsx

```diff
@@ -222,7 +222,7 @@ const ExplorerContextMenu = observer(
         applicationStore.navigator.openNewWindow(
           applicationStore.navigator.generateLocation(
             generateViewEntityRoute(
-              applicationStore.config.sdlcServerKey,
+              applicationStore.config.currentSDLCServerOption,
```

Comment:
Does this account for when you are viewing the explorer tree in gav mode. Seems like we are generating the viewr mode link with the default sdlc server option which is not necessarily right